### PR TITLE
iot and remote yaml pages

### PR DESCRIPTION
**Pull request by kenman345**
Check it!

### CODE REVIEW

File: _data/iot.yml [hunk 1]
```diff
@@ -4,41 +4,51 @@ websites:
...
+      btc: No
+      othercrypto: No
 
     - name: Nest
       url: https://nest.com
       img: nest.png
-      tfa: Yes
-      sms: Yes
-      doc: https://nest.com/support/article/How-to-use-2-step-verification-to-add-stronger-security-to-your-Nest-Account
+     bcc: No
```
Comment (rawb0t) [inline _data/iot.yml]: tab alignment.

File: _data/remote.yml [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
-      tfa: Yes
-      sms: Yes
-      software: Yes
-      doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_common_Security_TwoStep.html
-
+      bcc: No
+      btc: No
+      othercrypto: No
+      twitter: joinme
+
```
Comment (rawb0t) [inline _data/remote.yml]: im pedantic. spacing.

File: _data/remote.yml [hunk 3]
```diff
@@ -1,60 +1,56 @@
...
     - name: TeamViewer
       url: https://www.teamviewer.com
       img: teamviewer.png
-      tfa: Yes
-      software: Yes
-      doc: https://www.teamviewer.com/en/help/402-How-do-I-activate-deactivate-two-factor-authentication-for-my-TeamViewer-account.aspx
+      bcc: No
+      btc: No
+      othercrypto: No
+      twitter: teamviewer
```
Comment (rawb0t) [inline _data/remote.yml]: needs newline at end

**Comment by kenman345** [comment]
Reviewed changes, looks good.

**Pull request merged**